lib/oban/worker.ex

```diff
@@ -55,12 +55,15 @@ defmodule Oban.Worker do
 
   The value returned from `c:perform/1` can control whether the job is a success or a failure:
 
-  * `:ok` or `{:ok, value}` — the job is successful; for success tuples the `value` is ignored
+  * `:ok` or `{:ok, value}` — the job is successful; for success tuples the `value` is ignored.
+    The job is marked as `completed`.
 
   * `{:cancel, reason}` — cancel executing the job and stop retrying it. An error is recorded
-    using the optional `reason`, though the job is still successful.
+    using the optional `reason`. The job is marked as `cancelled`.
```

Comment:
It isn't actually optional:

```suggestion
    using the provided `reason`. The job is marked as `cancelled`.
```